src/convertToObject.js

```diff
@@ -6,7 +6,29 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  // debugger;
+
+  const getArrayFromString = sourceString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `array` , `object` `arr`, `obj`, `result`. It will decrease code readability. In this case, `getArrayFromString` could be more descriptive if named something like `styleDeclarations` or `styleEntries`.